dnf-behave-tests/features/scriptlets.feature

```diff
@@ -92,3 +92,106 @@ Scenario: Remove a pkg with a failing %postun scriptlet
    And Transaction is following
        | Action        | Package                           |
        | remove        | Package-postun-fail-0:1.0-1.x86_64 |
+
+
+Scenario: Correct output for triggered successful scriptlet of package not present in transaction
+  When I execute dnf with args "install Package-triggerin-ok"
```

Comment:
I think this is not part of the real test, it's the setup, so you can use:
```
Given I successfully execute dnf with args "install Package-triggerin-ok"
```

And then you don't need to check the exit status...